README.md

```diff
@@ -39,6 +39,7 @@ _Frameworks for building ASGI web applications._
 - [Channels](https://channels.readthedocs.io/en/latest/) - Asynchronous support for Django, and the original driving force behind the ASGI project. Supports HTTP and WebSockets with Django integration, and any protocol with ASGI-native code.
 - [Django](https://docs.djangoproject.com/en/3.0/topics/async/) - The web framework for perfectionists with deadlines. Has native ASGI support since version 3.0.
 - [FastAPI](https://github.com/tiangolo/fastapi) - A modern, high-performance web framework for building APIs with Python 3.6+ based on standard Python type hints. Powered by Starlette and Pydantic. Supports HTTP and WebSockets.
+- [Guillotina](https://github.com/plone/guillotina) - Guillotina is the only full-featured Python AsyncIO REST Resource Application Server designed for high-performance, horizontally scaling solutions.
```

Comment:
Suggestion to tune boilerplate down a bit and clarify how this relates to ASGI:

> Full-featured ASGI-compatible REST application framework, designed for high performance and horizontally scaling solutions.

Reply:
done :)